oaknational/Oak-Web-Application
# chore(update-gql): update the teacherLessonOverview schema to take mv…

Update teacherLessonOverview to point at mv_questions_2 and update sdk

## Description

- Changes GraphQL query to point at most recent quiz

## Issue(s)

Fixes #1420 

## How to test

1. Go to https://deploy-preview-1505--oak-web-application.netlify.thenational.academy/beta/teachers/key-stages/ks4/subjects/maths/units/quadratic-graphs-1-a-1-0f08/lessons/identify-and-interpret-roots-intercepts-and-turning-points-of-quadratic-graphs-c4uk8d
2. Click on _______
3. You should see _______

## Screenshots

How it used to look (delete if n/a):

![image](https://user-images.githubusercontent.com/91190841/231966978-6a715ee5-edbf-4ac9-a787-8b56eb5ee05d.png)


How it should now look:

![image](https://user-images.githubusercontent.com/91190841/231967018-6d625b77-b477-468c-8391-8a3e5c79c11e.png)


## Checklist
